ambry-api/src/main/java/com.github.ambry/config/StoreConfig.java

```diff
@@ -76,7 +76,14 @@
   public final int storeDeletedMessageRetentionDays;
 
   /**
-   * The rate of I/O allowed for compaction and hard deletes.
+   * The rate of I/O allowed for hard deletes.
+   */
+  @Config("store.hard.delete.operations.bytes.per.sec")
+  @Default("10*1024*1024")
+  public final int storeHardDeleteOperationsBytesPerSec;
+
+  /**
+   * The rate of I/O allowed for compaction.
    */
   @Config("store.cleanup.operations.bytes.per.sec")
```

Comment:
`store.compaction.operations.bytes.per.sec`? Cleanup is a more general term I think.

Reply:
sure. I also prefer `store.compaction.operations.bytes.per.sec`